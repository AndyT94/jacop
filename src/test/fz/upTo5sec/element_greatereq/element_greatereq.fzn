var 8..8: item:: output_var;
array [1..4] of var int: element_table:: output_array([1..4]) = [6,9,2,9];
solve  satisfy;
