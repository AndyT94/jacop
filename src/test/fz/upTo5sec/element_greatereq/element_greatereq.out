item = 8;
element_table = array1d(1..4, [6, 9, 2, 9]);
----------